dev/TabView/TabView.cpp

```diff
@@ -270,6 +270,60 @@ void TabView::OnSelectedIndexPropertyChanged(const winrt::DependencyPropertyChan
     SetTabSeparatorOpacity(winrt::unbox_value<int>(args.OldValue()) - 1);
     SetTabSeparatorOpacity(SelectedIndex() - 1);
     SetTabSeparatorOpacity(SelectedIndex());
+
+    UpdateTabBorderVisualStates();
+}
+
+void TabView::UpdateTabBorderVisualStates()
```

Comment:
I think it would be good to universally use the term BottomBorderline. So UpdateTabBottomBorderlineVisualStates

Reply:
Makes sense. I updated it everywhere.